controller_coverage.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Usage: bash ./controller_coverage.sh
+# Then run: firefox controller_coverage/index.html
+
+set -e
+set -x
+
+# delete coverage files
+#find .pio -name "*.gcno" -delete
+
+# run tests
+pio test -e native
+
+ENVIRONMENT=native
+OUTPUT_DIR=controller_coverage
```

Comment:
The root directory is getting cluttered with many ignored files, e.g. the graphs generated by the debug script, and now this.  This isn't your sin.  But at some point someone will need to clean it up, and e.g. move all of the locally ignored files into a single directory.  For you to consider.